xrtpy/response/tests/test_temperature_response.py

```diff
@@ -32,36 +32,38 @@ def test_temperature_response(filename):
         abundance = f.readline().split()[1]
         filter_name = f.readline().split()[1]
         filter_obs_date = " ".join(f.readline().split()[1:])
-    # NOTE: Annoyingly the date strings use "Sept" instead of "Sep" for "September"
+
+    # The IDL file might say "Sept" instead of "Sep"
     filter_obs_date = filter_obs_date.replace("Sept", "Sep")
+
     IDL_data = np.loadtxt(filename, skiprows=5)
     IDL_temperature = IDL_data[:, 0] * u.K
     IDL_temperature_response = IDL_data[:, 1] * u.Unit("DN cm5 pix-1 s-1")
 
     instance = TemperatureResponseFundamental(
-        filter_name, filter_obs_date, abundance_model=abundance
-    )
-
-    IDL_temperature_response = np.interp(
-        instance.CHIANTI_temperature, IDL_temperature, IDL_temperature_response
+        filter_name,
+        filter_obs_date,
+        abundance_model=abundance,
     )
 
     actual_temperature_response = instance.temperature_response()
 
+    IDL_temperature_response_interp = np.interp(
+        instance.CHIANTI_temperature.value,
+        IDL_temperature.value,
+        IDL_temperature_response.value,
+    ) * u.Unit("DN cm5 pix-1 s-1")
+
     # NOTE: there may be small deviations where the response function is very small, likely
     # due to differences in the interpolation schemes. These are not critical as the response
     # is effectively zero in these regions anyway.
     i_valid = np.where(
         actual_temperature_response > 1e-8 * actual_temperature_response.max()
     )
 
-    # NOTE: The relative tolerance is set comparatively high here because the CCD right gain
-    # values in the IDL and xrtpy codes are explicitly different. See https://github.com/HinodeXRT/xrtpy/pull/76.
-    # If the IDL results files are corrected to account for this updated gain, then this
-    # relative tolerance can be set significantly lower. Setting the gains to be the same,
-    # nearly all cases match within less than 1%.
     assert u.allclose(
         actual_temperature_response[i_valid],
-        IDL_temperature_response[i_valid],
-        rtol=5e-2,
+        IDL_temperature_response_interp[i_valid],
+        rtol=0.005,
+        atol=1e-6 * actual_temperature_response.max(),
```

Comment:
I'm wondering if it would be to reduce `rtol` to perhaps `0.001` if we increased the factor in `atol` to perhaps `1e-5`.  Mostly I'm thinking about making sure that the results are close to each other in places where the temperature response isn't negligibly small.